### DISCUSSION TITLE
Non managed prisma model

### DISCUSSION THREAD

**Discussion by VaZark** [prisma/prisma#13026] [Q&A]
Is there a way to ensure Prisma model has read only access to a db in the schema?

 I guess the easier way is to modify the db user access but being able to enforce that on prisma client would be a nice update.

**Comment by austincrim** [comment]
Hey @VaZark 👋 ,

Modifying the DB user's access would be the way to go here. Then you could use the read-only user's credentials to connect to the DB through Prisma.

If you have an alternative idea for how you want this to work, feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

**Reply by VaZark** [reply]
How feasible do you think it is to add a modifier to the prisma model that simply unsets all the functions except the select when generating the client??

**Reply by austincrim** [reply]
It should be possible using a custom generator. I would start by taking a look at other [community-built generators](https://www.prisma.io/docs/concepts/components/prisma-schema/generators#community-generators) to get a feel for the setup.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
